addons/binding/org.openhab.binding.freebox/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Freebox Binding
+Bundle-SymbolicName: org.openhab.binding.freebox;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/json-simple-1.1.jar,
+ lib/httpmime-4.3.1.jar,
+ lib/freeboxos-client-0.3.5.jar,
+ lib/jackson-core-asl-1.9.13.jar,
+ lib/jackson-mapper-asl-1.9.13.jar,
+ lib/httpclient-4.3.1.jar,
+ lib/httpcore-4.3.jar
+Import-Package: com.google.common.collect,
+ org.apache.commons.logging,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.openhab.library.tel.items,
```

Comment:
This import is actually not required and should be removed, right?
If you like to use the CallItem, feel free to create a PR to add it as a new item type in the CoreItemFactory in ESH.